use_cases/cacher.rb

```diff
@@ -8,6 +8,14 @@ def initialize(options={volume: nil, reporter: nil, page: page})
   end
 
   attr_accessor :volume, :reporter, :page, :cached_entry
+  
+  def self.cache_with_params!(params)
```

Comment:
I'm a noob at this ruby game. Is this _supposed_ to have `self` beforehand? Are the other methods just shorthands?